### PR TITLE
Optimise Subscription loader queries

### PR DESCRIPTION
**Pull request by danvixent**

### CODE REVIEW

File: database/postgres/subscription.go [hunk 1]
```diff
@@ -112,51 +113,36 @@ const (
...
-    select id, type, project_id, endpoint_id, function,
-    filter_config_event_types AS "filter_config.event_types",
-    filter_config_filter_headers AS "filter_config.filter.headers",
-	filter_config_filter_body AS "filter_config.filter.body"
-    from convoy.subscriptions
-    where (ARRAY[$4] <@ filter_config_event_types OR ARRAY['*'] <@ filter_config_event_types)
-    AND id > $1
-    AND project_id = $2
-    AND deleted_at is null
-    ORDER BY id LIMIT $3`
```
Comment (jirevwe) [inline database/postgres/subscription.go:125]: Why was this removed?

Comment (danvixent) [reply database/postgres/subscription.go:125]: We're not using it anymore

Comment (subomi) [reply database/postgres/subscription.go:125]: @danvixent We should leave it in; there's a potential use case for it to optimise the loader.

Comment (danvixent) [reply database/postgres/subscription.go:125]: @subomi okay

Comment (danvixent) [reply database/postgres/subscription.go]: Done

**Review by jirevwe — APPROVED** [review]
